go.mod

```diff
@@ -164,10 +164,10 @@ require (
 replace (
 	github.com/opencontainers/runc => github.com/opencontainers/runc v1.1.12
 	github.com/petermattis/goid => github.com/petermattis/goid v0.0.0-20240327183114-c42a807a84ba
-	golang.org/x/crypto => golang.org/x/crypto v0.19.0
+	golang.org/x/crypto => golang.org/x/crypto v0.21.0
```

Comment:
Since indirect require have a newer version, so @targetoee update replace golang.org/x/crypto to v0.21.0 as  I suggested.

Reply:
@wilfred-s, @craigcondit , @chia7712, 
I remember that we have concluded we should keep all the replace directive aligned across all the repositories. So I will request @targetoee to update replcae directive golang.org/x/crypto to v0.21.0 in core and shceduler-interface as well.

Reply:
Thanks for the reminder. I've updated the scheduler-interface first (https://github.com/apache/yunikorn-scheduler-interface/pull/136) and will update the core after.

Reply:
@targetoee Shim and core are updated, please update this PR with the latest version.